test/integration/framework/jsonify_benchmark.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	benchparse "golang.org/x/tools/benchmark/parse"
+	"k8s.io/kubernetes/test/e2e/perftype"
+)
+
+func main() {
+	err := run()
+	if err != nil {
+		panic(err)
```

Comment:
Do we indeed need to panic here? I worry that this may unnecessarily exit the benchmark script earlier (i.e without finishing the other tee for prettyprint) if this fails.

Reply:
Good point, this is indeed what would happen by default:

```
porridge@porridge-dell:~$ dd if=/dev/zero bs=1024 count=4096 | tee >(false) | wc -c
8192
porridge@porridge-dell:~$ dd if=/dev/zero bs=1024 count=4096 | tee >(cat > /dev/null) | wc -c
4096+0 records in
4096+0 records out
4194304 bytes (4.2 MB, 4.0 MiB) copied, 0.00709774 s, 591 MB/s
4194304
porridge@porridge-dell:~$ 
```

But there is a workaround:
```
porridge@porridge-dell:~$ dd if=/dev/zero bs=1024 count=4096 | tee -p  >(false) | wc -c
4096+0 records in
4096+0 records out
4194304
4194304 bytes (4.2 MB, 4.0 MiB) copied, 0.00840112 s, 499 MB/s
porridge@porridge-dell:~$ 
```

Reply:
Nice!